creating_gs_app.md

```diff
@@ -29,46 +29,72 @@ to put it, we have created a `guest_science_projects` folder for you to use.
 Please note, you don't have to use this directory and can store your project
 anywhere on your computer.
 
+**Important**: Since the Astrobee team is currently in the process of transitioning from 
+Ubuntu 16.04 to 20.04, there will be a divergence in the procedure to accomadate for both Ubuntu versions. Follow the guide according to your version and assume if there is no indication for which version it is for, that the steps work for both.
+
 ## Setup
 
 In every terminal you use, be sure to set up your environment. If you forgot how
 to do this, follow the Setting up your Environment section of the
-[`freeflyer/simulation/sim_overview.md`](https://github.com/nasa/astrobee/blob/master/simulation/sim_overview.md#setting-up-your-environment).
+[`astrobee/simulation/running_the_sim.md`](https://github.com/nasa/astrobee/blob/master/simulation/running_the_sim.md).
 
 ## Build the Astrobee API
 
 ### Generating ARS ROS Messages (`ff_msgs`)
 
-Ensure `rosjava` is installed:
-
-    you@machine:~ $ sudo apt-get install ros-kinetic-rosjava
+**Ubuntu 16.04**
 
+Ensure `rosjava` is installed:
+```shell
+  sudo apt-get install ros-kinetic-rosjava
+```
 Build the ff_msgs jar
-
-    you@machine:~ $ cd $BUILD_PATH
-    you@machine:native $ make rebuild_cache
-    you@machine:native $ make ff_msgs_generate_messages_java_gradle
-
+```shell
+  cd $BUILD_PATH
+  make rebuild_cache
+  make ff_msgs_generate_messages_java_gradle
+```
+**Ubuntu 20.04**
+
+As there is no ros-noetic-java for Ubuntu 20.04, the current solution is for
+developers to download the jar files here:
+
+<a href="https://github.com/nasa/astrobee/actions/workflows/msgs_jar.yaml?query=branch%3Adevelop"><img src="https://img.shields.io/badge/Build_jar_messages-passing-green.svg"/></a>
+ - Note: Be sure to download the most recent one that is passing.
+  
 ### Letting gradle access ARS messages
 
 The astrobee_api project is configured to look at the local maven repository
 to access the `ff_msgs` jar. The default location for this repository is:
-`$HOME/.m2/repository`. If you are building this project on the same machine
-that you build flight software on, the easiest thing to do is symlink the
+`$HOME/.m2/repository`. 
+```shell
+  mkdir -p $HOME/.m2
+  cd $HOME/.m2
+```
+**Ubuntu 16.04**
+
+If you are building this project on the same machine
+that you built the astrobeeunseen comment in github software on, the easiest thing to do is symlink the
```

Comment:
Sorry I missed this before, but I think there is an unexpected text inserted here.